installer/windows/Product.wxs

```diff
@@ -129,7 +129,7 @@
     <InstallExecuteSequence>
       <Custom Action='StoragenodeSetup' Before='InstallServices'>NOT Installed AND NOT WIX_UPGRADE_DETECTED</Custom>
       <Custom Action='DeleteConfigFile' After='RemoveFiles'>(REMOVE="ALL") AND NOT WIX_UPGRADE_DETECTED</Custom>
-      <!-- legacy: save config file as old versions of the installer will remove it -->
+      <!-- legacy: save config file as old versions or the installer will remove it -->
```

Comment:
It was really "of" :) Maybe "as older installer versions will remove it" would be clearer.

Reply:
:+1: